backend/app/views/offer_mailer/email_contract.html.erb

```diff
@@ -6,11 +6,19 @@
     </style>
   </head>
   <body>
+  <p><%= @status_message %></p>
+  <br>
   <p>Dear <%= @first_name %> <%= @last_name %></p>
   <p>
     We would like to offer you a position as a TA for <%= @position_code %> <%= @position_title %>.
   </p>
-  <p>
```

Comment:
Why did you move this to inside if? Whitespace doesn't matter at the start of a `<p>`. Also, are the tags closed correctly?